utest.h

```diff
@@ -33,6 +33,10 @@
 #ifndef SHEREDOM_UTEST_H_INCLUDED
 #define SHEREDOM_UTEST_H_INCLUDED
 
+#include <cstdlib>
```

Comment:
You are gonna run into some nasty cross platform issues here I'd bet!

Reply:
yes I think cross platform compatibility will be a problem. I will check if there is a solution for that.